apps/portal/src/components/pages/MagicLinkPage.js

```diff
@@ -180,20 +188,28 @@ export default class MagicLinkPage extends React.Component {
 
         return (
             <form onSubmit={e => this.handleSubmit(e)}>
-                <section className='gh-portal-section'>
+                <section className='gh-portal-section gh-portal-otp'>
                     {/* @TODO: create different input component with updated design */}
-                    <InputField
-                        id={`input-${OTC_FIELD_NAME}`}
-                        name={OTC_FIELD_NAME}
-                        type="text"
-                        value={this.state.otc}
-                        placeholder="• • • • • •"
-                        label={t('Code')}
-                        errorMessage={errors.otc || ''}
-                        autoFocus={false}
-                        maxLength={6}
-                        onChange={e => this.handleInputChange(e, {name: OTC_FIELD_NAME})}
-                    />
+                    <div className={`gh-portal-otp-field-container ${errors.otc && 'error'}`}>
+                        <input
+                            id={`input-${OTC_FIELD_NAME}`}
+                            className='gh-portal-input'
+                            name={OTC_FIELD_NAME}
+                            type="text"
+                            value={this.state.otc}
+                            placeholder="••••••"
+                            inputMode="numeric"
+                            pattern="[0-9]*"
+                            label={t('Code')}
```

Comment:
Should this be `aria-label`? Afaik `input` fields don't have a `label` attribute